src/api/helpers/get-populated-declaration/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import getDeclarationById from '../get-declaration-by-id';
+import getDeclarationModernSlaveryById from '../get-declaration-modern-slavery-by-id';
+
+/**
+ * getPopulatedDeclaration
+ * Get a populated declaration
+ * @param {Context} context: KeystoneJS context API
+ * @param {String} id: Declaration ID
+ * @returns {Promise<ApplicationDeclaration>}
+ */
+const getPopulatedDeclaration = async (context: Context, id: string) => {
+  try {
+    console.info('Getting populated declaration %s', id);
+
+    const declaration = await getDeclarationById(context, id);
+
```

Comment:
Should we validate `declaration` response?

Reply:
Good suggestion - I don't see a value in doing this though.

`getDeclarationById` calls and returns a KeystoneJS method which we have no control over - `context.db.Declaration.findOne`.

In this helper (`getPopulatedDeclaration`), we only consume `declaration.modernSlaveryId`, which we call `getDeclarationModernSlaveryById` with.

If `declaration.modernSlaveryId` doesn't exist or is an invalid ID etc, it'll throw an error in `getDeclarationModernSlaveryById`'s try/catch handler.